server/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM python:3.11 as lomas_server_base
 # To install missing packages
 COPY ./requirements.txt /requirements.txt
 RUN pip install --no-cache-dir --upgrade -r requirements.txt 
+ENV IN_CONTAINER=1
```

Comment:
putting the path instead of this variable created issues with file not found.. I really don't get it. I think it is fine like this but feel free to improve if you have better ideas (used to change path of configs and secrets in constants.py)